contrib/scalar/scalar.c

```diff
@@ -684,6 +684,9 @@ int cmd_main(int argc, const char **argv)
 		argv++;
 		argc--;
 
+		if (!strcmp(argv[0], "config"))
+			argv[0] = "reconfigure";
+
```

Comment:
Is this really what the old `scalar config` would do? I thought there was a Scalar-specific config file somewhere (based on the `gvfs config` command, which did things like set the custom upgrader details). Maybe just drop this one?

Reply:
Okay, I'll drop it.